src/Uno.UWPSyncGenerator/Generator.cs

```diff
@@ -528,14 +517,14 @@ List<ISymbol> writtenSymbols
 				var isDefinedInClass = ownerType.GetMembers().OfType<IMethodSymbol>().Any(m =>
 						m.Name == method.Name
 						&& m.DeclaredAccessibility == Accessibility.Public
-						&& m.Parameters.Select(p => GetDisplayStringWithoutNullableReferences(p.Type)).SequenceEqual(method.Parameters.Select(p2 => GetDisplayStringWithoutNullableReferences(p2.Type)))
-						&& GetDisplayStringWithoutNullableReferences(m.ReturnType) == GetDisplayStringWithoutNullableReferences(method.ReturnType)
+						&& m.Parameters.Select(p => p.Type.ToDisplayString()).SequenceEqual(method.Parameters.Select(p2 => p2.Type.ToDisplayString()))
```

Comment:
This was originally there to ensure that types would match regardless of nullability when comparing with WinRT, I'm not sure this is going to work.

Reply:
This just reverts my previous change where I added this `GetDisplayStringWithoutNullableReferences` method when trying to fix the generation issue in this PR. In the end I had to change `ToDisplayString` in a different place. But maybe it could be applied here too without issues, but I will just use the `ToDisplayString` overload which can handle nullable reference types.